client/src/Components/HomePage/MenuItemCategories/MenuItemCategory/CategorizedMenuItem/CategorizedMenuItem.jsx

```diff
@@ -1,5 +1,51 @@
 import React from "react";
+import "./CategorizedMenuItem.css";
+import AccordionDetails from "@mui/material/AccordionDetails";
+import AddShoppingCartIcon from "@mui/icons-material/AddShoppingCart";
+import { Button } from "@mui/material";
+import { useSelector, useDispatch } from "react-redux";
+import Login from "../../../../Login/Login";
+import { REQUEST_STATUSES } from "../../../../../constants/apiRequestStatus";
+import {
+  selectLoginStatus,
+  selectIsOpenDialog,
+  loginFormVisibilityChanged,
+} from "../../../../../slices/loginSlice";
 
-export default function CategorizedMenuItem() {
-  return <div>CategorizedMenuItem</div>;
+export default function CategorizedMenuItem(props) {
```

Comment:
I would prefer to props parameter as destructed object, for example `{ name, price }`, instead of just `props`. In this case at first look at the header of the component you can understand which props it accepts